<launch>
  <!-- Launch the ME5413 Path Publisher Node -->
  <node ns="me5413_world" pkg="me5413_world" type="path_publisher_node" name="path_publisher_node" output="screen" />
  <!-- Launch the ME5413 Path Publisher Node -->
  <node ns="me5413_world" pkg="me5413_world" type="lqr_node.py" name="path_tracker_node" output="screen" />
 <!-- <node ns="me5413_world" pkg="me5413_world" type="lqr_tracker_node" name="lqr_tracker_node" output="screen" /> -->
  <!-- Launch Rviz with our settings -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find me5413_world)/rviz/navigation.rviz" output="log" respawn="true"/>

  <!-- Dynamic Reconfigure GUI -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" />

<!-- <node name="record_bag" pkg="rosbag" type="record"
      args="-O /home/lando/ME5413_Planning_Project/error_lqr.bag /me5413_world/planning/abs_position_error /me5413_world/planning/abs_heading_error /me5413_world/planning/abs_speed_error 
      /me5413_world/planning/rms_heading_error /me5413_world/planning/rms_position_error /me5413_world/planning/rms_speed_error" />  -->


</launch>
